axelrod/strategies/cycler.py

```diff
@@ -1,13 +1,15 @@
-from axelrod.actions import Actions, Action
+from axelrod.actions import Actions, Action, str_to_actions
 from axelrod.player import Player
-
+import itertools
 import copy
 
+C, D = Actions.C, Actions.D
+
 
 class AntiCycler(Player):
     """
     A player that follows a sequence of plays that contains no cycles:
-    C CD CCD CCCD CCCCD CCCCCD ...
+    CDD  CD  CCD CCCD CCCCD ...
```

Comment:
this test in the link would fail on the original code.
[https://github.com/eric-s-s/Axelrod/blob/64ec1819521550cdeb4c7902313d3068da7f3a8e/axelrod/tests/strategies/test_cycler.py#L27-L35](url)